tools/pythonpkg/src/include/duckdb_python/pyrelation.hpp

```diff
@@ -192,6 +192,8 @@ struct DuckDBPyRelation {
 
 	duckdb::pyarrow::Table ToArrowTable(idx_t batch_size);
 
+	py::object ToPolars(idx_t batch_size);
```

Comment:
Can you create an object subclass for this instead, similar to what I did for the pandas DataFrame type?